incubator/artifactory/templates/deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: {{ template "fullname" . }}
-        image: "{{default "docker.bintray.io/jfrog/artifactory-oss" .Values.image}}:{{default "4.16.0" .Values.imageTag}}"
+        image: "{{default "docker.bintray.io/jfrog/artifactory-oss" .Values.image}}:{{default "5.1.0" .Values.imageTag}}"
```

Comment:
Let's drop the `default` statements here and use the `image` & `imageTag` values explicitly.